Xamarin.Forms.Platform.iOS/ExportRendererAttribute.cs

```diff
@@ -23,7 +23,10 @@ public ExportRendererAttribute(Type handler, Type target) : base(handler, target
 
 		public override bool ShouldRegister()
 		{
-			return !Idiomatic || Idiom == UIDevice.CurrentDevice.UserInterfaceIdiom;
+			if (!(!Idiomatic || Idiom == UIDevice.CurrentDevice.UserInterfaceIdiom))
```

Comment:
A little hard to parse this. Can we make it easier to understand without the double negative? :)